src/app/components/campus/modules/slideshow/IImage.ts

```diff
@@ -0,0 +1,10 @@
+export interface IImage {
```

Comment:
This whole `modules` folder looks like it was copied directly from the source of `ng-simple-slideshow`. Is there a reason it's here? The `SlideshowModule` that's being imported into the `AppModule` comes from the npm package, so I don't think any of this code is being used (from what I can tell, it's actually breaking the build). Could you remove it?